Amplify/Categories/DataStore/DataStoreCategory+HubPayloadEventName.swift

```diff
@@ -23,4 +23,15 @@ public extension HubPayload.EventName.DataStore {
     /// Dispatched when DataStore receives a sync response from the remote API via the API category. The Hub Payload
     /// will be a `MutationEvent` instance that caused the conditional save failed.
     static let conditionalSaveFailed = "DataStore.conditionalSaveFailed"
+
+    /// Dispatched on DataStore start and also every time a local mutation is enqueued and processed in the outbox
+    /// HubPayload `OutboxStatusEvent` contains a boolean value `isEmpty` to notify if there are mutations in the outbox
+    static let outboxStatus = "DataStore.outboxStatus"
+
+    /// Dispatched when all of the subscriptions to syncable models have been established
+    static let subscriptionsEstablished = "DataStore.subscriptionEstablished"
+
+    /// Dispatched when DataStore is about to start sync queries
```

Comment:
"Dispatched when DataStore starts establishing its subscriptions to all syncable models"

Reply:
For `syncQueriesStarted`, it is not `Dispatched when DataStore starts establishing its subscriptions to all syncable models but is `Dispatched when DataStore starts to sync Queries`